rpm/rpm.go

```diff
@@ -2,31 +2,27 @@ package rpm
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
+	"slices"
 	"sort"
 	"strings"
 
-	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
-	"github.com/aquasecurity/trivy/pkg/log"
-
-	"slices"
-
-	"github.com/aquasecurity/trivy/pkg/fanal/types"
-	"github.com/aquasecurity/trivy/pkg/fanal/utils"
 	rpmdb "github.com/knqyf263/go-rpmdb/pkg"
 	"github.com/samber/lo"
+	"golang.org/x/xerrors"
```

Comment:
Why do we need a third party package over the std lib one? Interface is exactly the same: `New` and `Errorf`. Both also support `Is` and `As` as well. 😕